source/operations/install-deploy-manage/deploy-operator-helm.rst

```diff
@@ -0,0 +1,447 @@
+.. _minio-k8s-deploy-operator-helm:
+
+=========================
+Deploy Operator With Helm
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+Helm is a tool for automating the deployment of applications to Kubernetes clusters.
+A `Helm chart <https://helm.sh/docs/topics/charts/>`__ is a set of YAML files, templates, and other files that define the deployment details.
+The following procedure uses a Helm Chart to install the :ref:`MinIO Kubernetes Operator <minio-operator-installation>` to a Kubernetes cluster.
+
+
+Prerequisites
+-------------
+
+To install the Operator with Helm you will need the following:
+
+* An existing Kubernetes cluster.
+* The ``kubectl`` CLI tool on your local host, the same version as the cluster.
+* `Helm <https://helm.sh/docs/intro/install/>`__ version 3.8 or greater.
+* `yq <https://github.com/mikefarah/yq/#install>`__ version 4.18.1 or greater.
+* Access to run ``kubectl`` commands on the cluster from your local host.
+
+For more about Operator installation requirements, including supported Kubernetes versions and TLS certificates, see the :ref:`Operator deployment prerequisites <minio-operator-prerequisites>`.
+
+This procedure assumes familiarity the with referenced Kubernetes concepts and utilities.
+While this documentation may provide guidance for configuring or deploying Kubernetes-related resources on a best-effort basis, it is not a replacement for the official :kube-docs:`Kubernetes Documentation <>`.
+
+
+Procedure
+---------
+
+
+Install Operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Download the Helm charts
+
+   On your local host, download the Operator and Tenant Helm charts to a convenient directory:
+
+   .. code-block:: shell
+      :class: copyable
+      :substitutions:
+
+      curl -O https://raw.githubusercontent.com/minio/operator/master/helm-releases/operator-|operator-version-stable|.tgz
+      curl -O https://raw.githubusercontent.com/minio/operator/master/helm-releases/Tenant-|operator-version-stable|.tgz
+
+   Each chart contains a ``values.yaml`` file you can customize to suit your needs.
+   For example, you may wish to change the MinIO root user credentials or the Tenant name.
+   For more about customizations, see `Helm Charts <https://helm.sh/docs/topics/charts/>`__.
+  
+#. Deploy Operator
+
+   The following Helm command deploys the MinIO Operator using the downloaded chart:
+
+   .. code-block:: shell
+      :class: copyable
+      :substitutions:
+
+      helm install \
+      --namespace minio-operator \
+      --create-namespace \
+      minio-operator operator-|operator-version-stable|.tgz
+
+#. Configure Operator
+
+   A. Create the YAML configuration files
```

Comment:
@cniackz just for my edification - are we just *persisting* the configurations to the namespace, as helm does not do that for us?